tests/util/misc.py

```diff
@@ -81,6 +81,17 @@ def zero_dataset() -> Dataset:
     return Dataset(tf.constant([[0.0]]), tf.constant([[0.0]]))
 
 
+def empty_dataset(query_point_shape: ShapeLike, observation_shape: ShapeLike) -> Dataset:
+    """
+    :param query_point_shape: The shape of a *single* query point.
+    :param observation_shape: The shape of a *single* observation.
+    :return: An empty dataset with points of the specified shapes, and dtype `tf.float64`.
+    """
+    qp = tf.zeros(tf.TensorShape([0]) + query_point_shape, tf.float64)
+    obs = tf.zeros(tf.TensorShape([0]) + observation_shape, tf.float64)
+    return Dataset(qp, obs)
+
+
 def one_dimensional_range(lower: float, upper: float) -> Box:
```

Comment:
What is this for?

Reply:
which bit? `empty_dataset`?

Reply:
Yep? Couldn't see where it was used. Is it just for testing?

Reply:
yeah I've used it in `test_efficient_global_optimization`. There are lots of tests elsewhere I'd use this too

Reply:
Makes sense then!